Description: this file is needed only when bash-completion is not available.
Forwarded: not-needed
Author: Tiago Ilieve <tiago.ilieve@gmail.com>

--- a/pythonpy/find_pycompletion.sh
+++ /dev/null
@@ -1,5 +0,0 @@
-py -c '
-import os
-import pkg_resources
-print(os.path.abspath(pkg_resources.resource_filename("pythonpy.completion", "pycompletion.sh")))
-'
--- a/setup.py
+++ b/setup.py
@@ -11,5 +11,4 @@
     long_description='https://github.com/Russell91/pythonpy',
     packages=['pythonpy', 'pythonpy.completion'],
     package_data={'pythonpy': ['completion/pycompletion.sh']},
-    scripts=['pythonpy/find_pycompletion.sh'],
 )
